plugin/evm/vm.go

```diff
@@ -37,6 +37,7 @@ import (
 	"github.com/ava-labs/coreth/params"
 	"github.com/ava-labs/coreth/peer"
 	"github.com/ava-labs/coreth/plugin/evm/message"
+	warpPrecompile "github.com/ava-labs/coreth/precompile/contracts/warp"
```

Comment:
We should force import module registry as well:
```
	// Force-load precompiles to trigger registration
	_ "github.com/ava-labs/coreth/precompile/registry"
```

Reply:
Updated